lib/box-node-sdk.js

```diff
@@ -117,6 +117,36 @@ BoxSDKNode.accessLevels = BoxSDKNode.prototype.accessLevels = BoxClient.prototyp
 BoxSDKNode.collaborationRoles = BoxSDKNode.prototype.collaborationRoles = BoxClient.prototype.collaborationRoles;
 BoxSDKNode.CURRENT_USER_ID = BoxSDKNode.prototype.CURRENT_USER_ID = BoxClient.prototype.CURRENT_USER_ID;
 
+BoxSDKNode.getInstance = function(appConfig) {
```

Comment:
I would add something to this method name to indicate what it's used for; this isn't really the "primary" way to get an instance, since it requires a special config object from the Dev Console